docs/components/src/component.md

```diff
@@ -0,0 +1,199 @@
+# InputFacade
+
+> The component is basically a wrapper around a native input element, as such it inherits all
+> properties available to [HTMLInputElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement).
+
+However it provides a cleaner and more straight forward interface to the directive's features.
+
+## Props
+
+| Prop name | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Type           | Values | Default |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- | ------ | ------- |
+| formatter | A function to format the value after applying the mask. The function will receive an<br/>object with the masked and unmasked value. The result of this function will determine<br/>what happens with the value.<br/><br /><br/>If a string is returned, then that string will pass through the masker function once more and its value<br/>will be set to the input. If false (boolean) is returned, the input will be rejected and the<br/>previous value will be restored. Otherwise the facade logic will continue as usual.<br/>`@since` v1.3 | func           | -      |         |
+| lazy      | Vue's v-model .lazy modifier does not currently work with custom components. If you wish to have your v-model<br/>updated only during the change event instead of on input, enable this property. <b>Note: This works by supressing<br/>input events and only emitting a single input event at the same time as the change event.</b><br/>`@since` v1.3                                                                                                                                                                                           | boolean        | -      | false   |
```

Comment:
Do we need to re-run the docgen or something? You removed this `lazy` prop didn't you?

- [ ] Investigate

Reply:
good catch. docgen was something I added since styleguidist does not work with vue3 at the moment. I am open to the team's input if this is helpful or redundant if we can just view the existing markdown without styleguidist. There have been updates made for vue3 compatibility in the [4.46.0 release](https://github.com/vue-styleguidist/vue-styleguidist/releases/tag/vue-styleguidist%404.46.0) but they are not yet available through npm.

Reply:
Hm actually does seem pretty redundant. I personally find it more clunky to read than just looking at the source code or .md file directly. I'm in favor of removing it.